src/sentry/web/frontend/debug/mail.py

```diff
@@ -385,33 +385,18 @@ def release_alert(request):
             "rules": get_rules([rule], org, project),
             "group": group,
             "event": event,
+            "event_user": event.data["user"],
```

Comment:
Do you need to update `sentry/notifications/notifications/rules.py` `ActiveReleaseAlertNotification.get_context` to include the extra context details here?

Reply:
Updated 👍